### DISCUSSION TITLE
Any way to download a single example without cloning the whole library?

### DISCUSSION THREAD

**Discussion by johhansantana** [blitz-js/blitz#1745] [Q&A]
Title question  😅

**Comment by flybayer** [comment]
Are you talking about the apps in `examples/` folder of the main repo?

**Reply by johhansantana** [reply]
Yeah. Sorry for the newbie question but I've seen some examples in stackoverflow on how to do it but they all seem overcomplicated. Not sure if you have a preferred way 😕

**Reply by flybayer** [reply]
Ok then yeah I don't know! Stackoverflow is probably the best way

**Reply by kvnwolf** [reply]
@flybayer what if we change a little bit the `blitz create` command to be able to clone an example from the `examples` folder instead of setting up a generic one? This can come with many benefits:

1. If I want to start an application without authentication, I can just pick a "basic" example.
2. The `generator` package will be simplified because we don't need to put auth code in there. Instead, we can create a `with-auth` example.
3. Users could choose actually what they want to start with, and, for instance, kick off an e-commerce from an `e-commerce` example.

**Reply by flybayer** [reply]
@kevinwolfcr I'm definitely in favor of moving towards a more "piece-meal" approach to app generation, but I feel pretty strongly that recipes are the way to do that instead of examples. 

We want minimize examples as much as possible because they are hard to maintain, keep up to date, etc.
